### PR TITLE
feat(docs): Add Google Service Account guide

### PR DESCRIPTION
**Pull request by wrussell1999**
Might add a quick video

### CODE REVIEW

File: content/docs/15.how-to-guides/google-credentials.md [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+```
+
+Once you've done that, copy and paste the response as a new environment variable, that starts with `SECRET_`, in your `docker-compose.yml`:
+```yaml
+kestra:
+  environment:
+    SECRET_GCP_SA: base64_encoded_json
+```
+
+## Add Secrets to a `.env` file
```
Comment (shrutimantri) [inline content/docs/15.how-to-guides/google-credentials.md:67]: Should we add a section on how to add secret for EE as we do not need to encode in that case?

File: content/docs/15.how-to-guides/google-credentials.md [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+mimeType: "application/vnd.google-apps.spreadsheet"
+serviceAccount: |
+  {
+    "type": "service_account",
+    "project_id": "...",
+    "private_key_id": "...",
+    "private_key": "...",
+    "client_email": "...",
+    "client_id": "...",
+    "auth_uri": "https://accounts.google.com/o/oauth2/auth", "token_uri": "https://oauth2.googleapis.com/token"
```
[the remark this answers is not in the document]
Comment (wrussell1999) [reply content/docs/15.how-to-guides/google-credentials.md:39]: good spot!

File: content/docs/15.how-to-guides/google-credentials.md (hunk 1 above)
Comment (wrussell1999) [reply content/docs/15.how-to-guides/google-credentials.md:68]: agreed - this is a good opportunity to show the benefits of EE secrets too

File: content/docs/15.how-to-guides/google-credentials.md [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+## Set the Service Account with `PluginDefaults`
+
+If you're using multiple tasks that will require the service account secret, you can set up a Plugin Default to apply this property to all tasks of this type. For example:
+```yaml
+tasks:
+  - id: "upload"
+    type: "io.kestra.plugin.googleworkspace.drive.Upload"
+    from: "{{ inputs.file }}"
+    parents:
+    - "1HuxzpLt1b0111MuKMgy8wAv-m9Myc1E_"
```
Comment (shrutimantri) [inline content/docs/15.how-to-guides/google-credentials.md:129]:
```suggestion
      - "1HuxzpLt1b0111MuKMgy8wAv-m9Myc1E_"
```

**Pull request merged**